setup.py

```diff
@@ -24,7 +24,7 @@
     url='https://github.com/pcmanus/ccm',
     packages=['ccmlib', 'ccmlib.cmds'],
     scripts=[ccmscript],
-    install_requires=['pyYaml', 'psutil', 'six >=1.4.1', 'requests'],
+    install_requires=['pyYaml', 'psutil', 'six >=1.4.1', 'requests', 'docker'],
```

Comment:
this was already addressed in #263 or if this one will be merged first... chat with @Orenef11 to avoid duplicate work